Cargo.toml

```diff
@@ -33,12 +33,15 @@ snafu = "0.6.8"
 hts-sys = { version = "^1.10", path = "hts-sys", default-features = false }
 
 [features]
-default = ["bzip2", "lzma", "curl"]
+default = ["bzip2", "lzma", "curl", "bindgen"]
 bzip2 = ["hts-sys/bzip2"]
 lzma = ["hts-sys/lzma"]
+bindgen = ["hts-sys/bindgen"]
 curl = ["hts-sys/curl"]
+libdeflate = ["hts-sys/libdeflate-sys"]
+s3 = ["hts-sys/s3"]
+gcs = ["hts-sys/gcs"]
 serde = ["serde_base", "serde_bytes"]
-static = []
```

Comment:
@pmarks I just noticed this... all builds will be `static` by default now? I'm fine with since I need MUSL builds to be static but perhaps folks want to i.e link against a modified locally installed `zlib`, for instance?

Reply:
agreed, we should retain. fixed.